src/ComputationContainer/Random/Csprng.cpp

```diff
@@ -77,12 +81,13 @@ std::vector<long long int> CSPRNG::GetRandLLVec(unsigned int size)
 {
     std::vector<long long int> randLLVec = {};
 
-    constexpr unsigned int LL_SIZE = sizeof(long long int);  // 8byte
-    const unsigned int byteSize = size * LL_SIZE;            // size * 8[byte/llsize]
+    constexpr std::size_t LL_SIZE = sizeof(long long int);  // 8byte
+    const std::size_t byteSize = size * LL_SIZE;            // size * 8[byte/llsize]
 
-    unsigned char *rnd = new unsigned char[byteSize];
+    // std::make_shared<unsigned char[]> is available in C++20
+    std::unique_ptr<unsigned char[]> rnd = std::make_unique<unsigned char[]>(byteSize);
     // 64bit乱数[unsigned char*]生成
-    this->CSPRNG::GetRand(rnd, byteSize);
+    this->CSPRNG::GetRand(rnd.get(), byteSize);
 
     // unsiged char* -> str(bin)
     for (unsigned int i = 0; i < byteSize; i += LL_SIZE)
```

Comment:
This is not essence, but since the change to `size_t` has been made elsewhere, Could you also change this?

Reply:
Noted with thanks.  I missed that.